### PR TITLE
fixes super sayan speed modular armor

### LINKED ISSUE #4299: Able to repeatedly remove jaegar armor parts for negative armor and SUPER FAST MOVEMENT
## Reproduction:

Apply armor to jaegar, spam crowbar it out while it's on the ground.
Marvel at your -300% armor and Sanic movement speed.

### PR DESCRIPTION
**Pull request by MacBlaze1**
## About The Pull Request

This PR fixes #4299
Closes #4299 
## Why It's Good For The Game

critical fix bugs bad

## Changelog
:cl:
fix: fixed the ability to gain super speed via exploiting detaching 1 armor piece from modular armor
/:cl:

### CODE REVIEW

File: code/modules/modular_armor/modular.dm [hunk 1]
```diff
@@ -202,6 +202,9 @@
 	if(.)
 		return
 
+	if(user.action_busy)
+		return FALSE
```
Comment (psykzz) [inline code/modules/modular_armor/modular.dm:206]: add this to each?

Comment (MacBlaze1) [reply code/modules/modular_armor/modular.dm:206]: Each what

Comment (psykzz) [reply code/modules/modular_armor/modular.dm:206]: each `x_act`, since the issue will happen with each of them no?

**Comment by GravitronBlue** [comment]
Exploit can still be done by queing up prompts by there being two armor pieces in the exosuit and crowbaring, then selecting the same piece repeatedly to remove it.

**Pull request merged**